src/gpio.rs

```diff
@@ -173,8 +204,6 @@ macro_rules! gpio {
             }
 
             impl AFRH {
-                // TODO remove `allow`
-                #[allow(dead_code)]
```

Comment:
I guess this was a leftover from when `AFRH` was not used? It does not give me any warnings, so I removed it.